airqo-platform/AirQo-api
# Ft join platform

- What does this PR do?
This PR created new endpoints for registering a candidate, accepting user join request, denying user JOIN request. Same PR fixed some bugs that were present in the password reset feature.
- What JIRA task is related to this PR?
The link to the JIRA board task is [here](https://airqoteam.atlassian.net/browse/NLYTCS-27).
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/auth-service` Then run `npm install` to get all the necessary dependencies Afterwards run `npm start` to test out the end points using any REST client tool available to you.
- Which new end points should are ready for testing?
[AirQo RESTful API documentation](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit?usp=sharing)
1. [POST] partial url: /api/v1/users/accept/:USER_ID
2. [POST] partial url: /api/v1/users/deny/:USER_ID
